geode-assembly/src/integrationTest/resources/assembly_content.txt

```diff
@@ -1234,6 +1234,8 @@ lib/jgroups-3.6.14.Final.jar
 lib/jline-2.12.jar
 lib/jna-4.1.0.jar
 lib/jopt-simple-5.0.4.jar
+lib/lombok-1.16.20.jar
+lib/lombok-1.18.4.jar
```

Comment:
Probably only need only of these right?

Reply:
Also, since Lombok is MIT licensed you'll need to reference it in the LICENSE in geode-assembly.

Reply:
I think 1.16.20 is used by the gradle-lombok plugin.